src/04-modality-specific-files/10-eye-tracking.md

```diff
@@ -0,0 +1,187 @@
+
+Eye-tracking data MUST be stored in the main data recording modality or `<datatype>` directory (for example func, anat, dwi, meg, eeg, ieeg, or beh). The current version of this extension *does not* include a specification of the extension format and data formatting of recorded eye-tracking data. Thus, data must be stored in an open format (for example EDF file converted in ASCII `.asc` format or `.tsv` etc) with the `_eyetrack` suffix.
+
+We encourage every user to put the raw data before conversion within the `/sourcedata` folder (for example put property EDF files in the sourcedata folder).
+
+The OPTIONAL [`task-<label>`](../99-appendices/09-entities.md#task) is used to indicate a task subject were asked to perform while eye-tracking records were obtained. Those labels MUST be consistent across subjects and sessions. For task based eye-tracking, a corresponding [task events](../04-modality-specific-files/05-task-events.md) file MUST be provided (please note that this file is not necessary for resting state tasks).
+
+The OPTIONAL [`acq-<label>`](../99-appendices/09-entities.md#acq) refers to a custom label the user MAY use to distinguish a different set of parameters used for acquiring the same modality. Acquisition labels corresponds mostly to imaging records (for example eye-tracking with fMRI) but can also be specified when combining eye-tracking and imaging methods.
+
+If more than one run of the same task and acquisition are acquired during the same session, the [`run-<index>`](../99-appendices/09-entities.md#run) entity MUST be used: _run-1, _run-2, _run-3, and so on. If only one run was acquired the run-<index> can be omitted.
+
+### Sidecar JSON document (`*_eyetrack.json`)
+
+Generic fields MUST be present:
+
+{{ MACROS___make_metadata_table(
+   {
+      "TaskName": ("REQUIRED", "A RECOMMENDED convention is to name resting state task using labels beginning with `rest`."),
+   }
+) }}
+
+SHOULD be present: 
+
+For consistency between studies and institutions, we encourage users to extract the values of these fields from the actual raw data. Whenever possible, please avoid using ad hoc wording.
+
+{{ MACROS___make_metadata_table(
+   {
+      "InstitutionName": "RECOMMENDED",
+      "InstitutionAddress": "RECOMMENDED",
+      "Manufacturer": (
+         "RECOMMENDED",
+         "Manufacturer of the eye-tracking system (for example "
+         '`"SR-Research"`, `"Tobii"`, `"SMI"`, `"Gazepoint"`, '
+         '`"Pupil Labs"`, `"Custom built"`, `"Other"`) '
+      ),
+      "ManufacturersModelName": (
+         "RECOMMENDED",
+         "Manufacturer's designation of the eye-tracker model (for example"
+         '`"Eye-link 1000"`).'
+         ),
+      "SoftwareVersion": "RECOMMENDED",
+      "TaskDescription": "RECOMMENDED",
+      "Instructions": (
+         "RECOMMENDED", 
+         "Text of the instructions given to participants before the experiment."),
+      "CogAtlasID": "RECOMMENDED",
+      "CogPOID": "RECOMMENDED",
```

Comment:
Should probably only be included if this is standalone eyetracking data and should otherwise be documented in the json of the data the eyetracking data is associated with.